### DISCUSSION TITLE
Share VPN connection via HTTP/HTTPS and Socks proxy in network

### DISCUSSION THREAD

**Discussion by majicson** [vpnhood/VpnHood#130] [Ideas]
Hi everybody
If we can share it on network and use it with other devices it's so good.
thanks

**Comment by bobvhood** [comment]
Yap. I believe it too. I hope we have time to implement that.

**Reply by Abdalhalanz45** [reply]
https://github.com/casebell/VpnHood/tree/main

**Reply by trudyhood** [reply]
> https://github.com/casebell/VpnHood/tree/main

Any comment?
